src/services/dashboards/resource.js

```diff
@@ -7,12 +7,12 @@ import {
   ActivityReportObjective,
   ActivityRecipient,
   Grant,
-  NextStep,
+  // NextStep,
```

Comment:
they were unused but are linked to the huge mass of commented code in this file, so I think we should leave them in there since it matches the other commented out stuff.